src/bio/bio_xstream.c

```diff
@@ -159,6 +159,25 @@ bypass_health_collect()
 	return nvme_glb.bd_bypass_health_collect;
 }
 
+struct bio_faulty_criteria	glb_criteria;
+
+/* TODO: Make it configurable through control plane */
+static inline void
+set_faulty_criteria(void)
+{
+	glb_criteria.fc_enabled = false;
+	glb_criteria.fc_max_io_errs = 5;
+	glb_criteria.fc_max_csum_errs = 5;
```

Comment:
There is a user settable pool property (DAOS_PROP_PO_SCRUB_THRESH) that the checksum scrubber as a threshold. Wondering if we should use the same property here instead of hard coding it? Could be in a future patch.

Reply:
Yes, all these hardcoding numbers are not for the production feature. We have to make the faulty criteria configurable through control plane (probably not per-pool but a global configuration?) when we officially support auto device faulty.